lib/KIcon/iconDefinitions.js

```diff
@@ -129,6 +129,9 @@ const KolibriIcons = {
   quiz: {
     icon: require('./precompiled-icons/mdi/file-document-edit-outline.vue').default,
   },
+  exam: {
```

Comment:
I think that I removed this altogether based on suggestion from @jtamiace - are there any missing icons in Kolibri that were found?

Whenever I removed a key from this file, I made corresponding changes in Kolibri (or tried to - if there are things trying to use the `exam` key still)

Reply:
okay awesome, will remove